src/Network/Broadcast/OutboundQueue.hs

```diff
@@ -232,12 +253,63 @@ defaultEnqueuePolicy NodeEdge = go
         -- not forwarded
       ]
     go MsgRequestBlockHeaders = [
-        EnqueueAll NodeRelay (MaxAhead 20) PLowest
+        EnqueueAll NodeRelay (MaxAhead 1) PHigh
```

Comment:
This should be max ahead 0. Since we don't send header announcements then those are not expected to be ahead of us (as they are in the case of other node types).